src/engine/x86-64/X86_64Runtime.v3

```diff
@@ -106,6 +106,11 @@ component X86_64Runtime {
 		if (ret != null) return stack.throw(ret);
 		return ret;
 	}
+	def runtime_materialize_frame_accessor() {
```

Comment:
How about calling this `runtime_getFrameAccessorMetaRef()`and returning the metaref, since it's needed by the caller?